app/tornado_handlers/download.py

```diff
@@ -60,79 +69,71 @@ def get_original_filename(default_value, new_file_suffix):
                 print("DB access failed:", sys.exc_info()[0], sys.exc_info()[1])
             return default_value
 
-
-        if download_type == '1': # download the parameters
+        if download_type == "1":  # download the parameters
             ulog = load_ulog_file(log_file_name)
             param_keys = sorted(ulog.initial_parameters.keys())
 
             self.set_header("Content-Type", "text/plain")
-            self.set_header('Content-Disposition', 'inline; filename=params.txt')
+            self.set_header("Content-Disposition", "inline; filename=params.params")
+
+            vehicle_id = 1  # You can set this to the appropriate vehicle ID
+            component_id = 1  # You can set this to the appropriate component ID
+            version = "1.14.0 beta"  # Set the version accordingly
+            git_revision = "0099523710000000"  # Set the Git revision accordingly
+
+            self.write(f"# Onboard parameters for Vehicle {vehicle_id}\n")
+            self.write("#\n")
+            self.write(f"# Stack: PX4 Pro\n")
+            self.write("# Vehicle: Multi-Rotor\n")
+            self.write(f"# Version: {version}\n")
+            self.write(f"# Git Revision: {git_revision}\n")
```

Comment:
Can you get these from the log or remove altogether? I prefer having no info over something incorrect.